cluster-logging/overlays/moc/configmaps/curator.yaml

```diff
@@ -0,0 +1,117 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: curator
+data:
+  actions.yaml: >
+    # ---
+
```

Comment:
can you please remove all those empty lines, please? It just bloats the resource...

Reply:
The blank lines are there because the content is using the folding block operator (`>`) instead of the literal block operator (`|`). 

This:

```
value: >
  one

  two

  three
```

Is equivalent to:

```
value: |
  one
  two
  three
```

Probably best just to use the literal block operator instead:

```
data:
  actions.yaml: |
    # ---
    # Remember, leave a key empty if there is no value. None will be a string...
```

Reply:
updated :+1: